.tekton/plnsvc-static-code-analysis.yaml

```diff
@@ -0,0 +1,107 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: plnsvc-static-code-analysis
```

Comment:
Use the full name if there's no length restriction (and rename the file to match).

Reply:
fixed.

Reply:
I think what @Roming22 meant to say was expand plnsvc to pipeline-service. 
It would be better to keep a generic name instead of mentioning the tools.
Please update it to 'pipeline-service-static-code-analysis'

Reply:
got it, thanks. fixed.